compose/lite/configuration.yml

```diff
@@ -0,0 +1,91 @@
+###############################################################
+#                   Authelia configuration                    #
+###############################################################
+
+host: 0.0.0.0
+port: 9091
+
+logs_level: debug
+
+# This secret can also be set using the env variables AUTHELIA_JWT_SECRET
+jwt_secret: a_very_important_secret
+
+default_redirection_url: https://auth.example.com
+
+totp:
+  issuer: authelia.com
+
+duo_api:
+  hostname: api-123456789.example.com
+  integration_key: ABCDEF
+  # This secret can also be set using the env variables AUTHELIA_DUO_API_SECRET_KEY
+  secret_key: 1234567890abcdefghifjkl
+
+authentication_backend:
+  file:
+    path: /etc/authelia/users_database.yml
+
+access_control:
+  default_policy: deny
+
+  rules:
+    # Rules applied to everyone
+    - domain: auth.example.com
+      policy: bypass
+
+    - domain: traefik.example.com
+      policy: one_factor
+
+    - domain: secure.example.com
+      policy: two_factor
+
+session:
+  name: authelia_session
+
+  # This secret can also be set using the env variables AUTHELIA_SESSION_SECRET
+  secret: unsecure_session_secret
+
+  expiration: 3600 # 1 hour
+
+  inactivity: 300 # 5 minutes
+
+  domain: example.com # This should match whatever your root protected domain is
+
+  redis:
+    host: redis
+    port: 6379
+    # This secret can also be set using the env variables AUTHELIA_SESSION_REDIS_PASSWORD
+    password: authelia
+
+regulation:
+  max_retries: 3
+
+  find_time: 120
+
+  ban_time: 300
+
+storage:
+  local:
+    path: /var/lib/authelia/db.sqlite3
+
+notifier:
+  smtp:
+    username: test
+    # This secret can also be set using the env variables AUTHELIA_NOTIFIER_SMTP_PASSWORD
+    password: password
+    host: 127.0.0.1
+    port: 1025
+    sender: admin@example.com
+    ## disable_require_tls: false
```

Comment:
Remove the commented options, they don't bring anything. For advanced use cases, users will have the documentation to review the entire set of options.